### DISCUSSION TITLE
Exclude client or domain from statistics

### DISCUSSION THREAD

**Discussion by julianpomper** [AdguardTeam/AdGuardHome#2878] [Ideas]
I do not want to see requests of some devices in my statistics because they for example are just NTP requests and "spam"/"distort" the statistics.
It would be cool to have an option to exclude a client or domain from the statistics.

**Comment by ameshkov** [comment]
I think there's already a similar feature request somewhere, seems to be rather popular: https://github.com/AdguardTeam/AdGuardHome/issues/1717

**Reply by julianpomper** [reply]
oh sorry, how did I miss that 🤔  
Yeah this would be cool! Should I close this discussion?

**Reply by ameshkov** [reply]
I guess so since it's a duplicate:)

**Reply by julianpomper** [reply]
sorry, I have no option to delete or mark it as "answered"

**Reply by ameshkov** [reply]
No worries, let it be then :)
